src/apps/explorer/components/TokensTableWidget/index.tsx

```diff
@@ -71,39 +79,52 @@ const ExtraComponentNode: React.ReactNode = (
     <TablePagination context={TokensTableContext} fixedResultsPerPage />
   </WrapperExtraComponents>
 )
+interface Props {
+  networkId: BlockchainNetwork
+}
```

Comment:
nit: can you give a not so generic name for these props? 
you use it in `TokensTableWidget` so what about `TokensTableWidgetProps` and export it so other components can use it

Reply:
also, see next lines. Looks like a duplicate